TAScheduler/AcceptanceTests/admin_create_course_test.py

```diff
@@ -0,0 +1,60 @@
+from django.test import TestCase
+from django.urls import reverse
+from TAScheduler.models import User, Course, Instructor, TA, Administrator
+
+
+class AdminCreateCourseTestCase(TestCase):
+    def setUp(self):
+        # Create an admin user for login
+        admin_user = User.objects.create(
+            email_address='admin@example.com',
+            password='adminpassword',
+            first_name='Admin',
+            last_name='User',
+            home_address='123 Admin St',
+            phone_number='1234567890'
+        )
+        Administrator.objects.create(user=admin_user)
+        self.client.login(email_address=admin_user.email_address,
+                          password='adminpassword')  # Update as per your login logic
+
+    def test_create_course_success(self):
+        # Test that admin can create a new course successfully
+        response = self.client.post(reverse('create_course'), {
+            'course_id': 101,
+            'semester': 'Fall 2023',
+            'name': 'Introduction to Testing',
+            'description': 'A course about writing tests in Django.',
+            'num_of_sections': 3,
+            'modality': 'Online',
+            'credits': 4
+        })
+        self.assertEqual(response.status_code, 302)  # Assuming redirection after successful creation
+        self.assertTrue(Course.objects.filter(name='Introduction to Testing').exists())
+
```

Comment:
I would recommend filtering using the and not the name. It may not be complete yet, but there are no requirements that the name's between courses are unique (as long as the ID's are unique). We could technically have 2 introduction to testing classes, with 2 separate ID's, and that's fine (in my opinion - We could add another AC specifically for this but it seems a little niche for our project) - Please let me know what you think: should we be testing for unique names too? (if yes, we would also need to test for unique names across TAs,Instructors, and all other named objects)
maybe another test to test the course field's saved in the Course object. 

For example:
This test just wants to see if there exists a course with "Introduction to Testing" ... we should take it a step further to say "there exists a course with "Introduction to Testing"  A N D all the fields are the same as our input"